package.json

```diff
@@ -32,18 +32,20 @@
   "author": "Lukas Aichbauer  <l.aichbauer@gmail.com> (https://github.com/aichbauer/)",
   "license": "MIT",
   "dependencies": {
-    "babel-core": "^6.23.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.2.1",
-    "babel-register": "^6.23.0",
-    "express": "^4.15.2",
+    "@babel/cli": "^7.0.0",
```

Comment:
We should move the `@babel/*` dependencies to devdependencies except of `@babel/runtime`.

Reply:
I have tried to move that `@babel/*` dependencies to devDependencies, but it will give an error when execute `npm run test`

Reply:
Try moving these to devDependencies:

```json
    "@babel/cli": "^7.0.0",
    "@babel/core": "^7.0.0",
    "@babel/plugin-transform-runtime": "^7.0.0",
    "@babel/preset-env": "^7.0.0",
```

and keep this as dependencies:

```json
    "@babel/register": "^7.0.0",
    "@babel/runtime": "^7.0.0",
```